operator/src/main/java/oracle/kubernetes/operator/helpers/CrdHelper.java

```diff
@@ -597,6 +613,15 @@ public NextAction onSuccess(
         LOGGER.info(MessageKeys.CREATING_CRD, callResponse);
         return doNext(packet);
       }
+
+      @Override
+      protected NextAction onFailureNoRetry(Packet packet, CallResponse<V1CustomResourceDefinition> callResponse) {
```

Comment:
New functionality without a unit test

Reply:
Same as above, it's resiliency bug fix to allow operator to start in case of missing permissions. I have added whenReplaceFails_scheduleRetryAndLogFailedMessageInOnFailureNoRetry() unit test to cover this.

Reply:
I have added a new unit test "whenReplaceFailsThrowsStreamException_scheduleRetryAndLogFailedMessageInOnFailureNoRetry"  for StreamResetException check .